# loads a plugin

# absolute path of plugin recommended
string filename
---
